src/commands/train.ts

```diff
@@ -0,0 +1,28 @@
+import { CommandType } from "../types";
+import { logger } from "../logger";
+import { SlashCommandBuilder } from "@discordjs/builders";
+import { CommandInteraction, CacheType } from "discord.js";
+import { dog, goat, train } from "../helpers/ASCIIArts";
+
+const pongModule: CommandType = {
+  data: new SlashCommandBuilder()
+    .setName("train")
+    .setDescription("Train?"),
+  execute: async (interaction: CommandInteraction<CacheType>) => {
+    try {
+
+      let codeBlockAdded = "```" + train + "```"
+      // let codeBlockAdded2 = "```" + goat + "```"
```

Comment:
is it possible to have 3 different commands for train, goat, and dog? maybe `/train`, `/goat`, and `/woof` or something.

Reply:
Yeb, but do you want it to be in a dropdown menu in one command only?

Reply:
I am fine with either. If we do the route of subcommands, we should have the base command be something similar to `/print` or `/art` or something